lib/pigeon/notification.ex

```diff
@@ -52,6 +52,8 @@ defmodule Pigeon.APNS.Notification do
 
   def put_priority(notification, priority), do: %{notification | priority: priority}
 
+  def put_ttl(notification, ttl), do: %{notification | time_to_live: ttl}
```

Comment:
APNS doesn't seem to support this at all. Can you share here a link to documentation where you found it? If you haven't, please confirm that it's indeed not possible to get this behavior with APNS ale let's drop it here - there's no point in sending something that will be simply ignored, not to mention that is misleading.